docs/index.bs

```diff
@@ -88,6 +88,12 @@ spec: ecma-262; urlPrefix: https://tc39.es/ecma262/
             text: statement
             text: declaration
 
+spec: html; urlPrefix: https://html.spec.whatwg.org/
+    type: dfn
+        text: HTML Standard;
+        text: set up; url:set-up-a-window-environment-settings-object
```

Comment:
I understand that you do not want to write a long name to represent it inside the document.  However, I feel referring this with [=set up=] too ambiguous.  Can this be [=set up a window environment settings object=] instead?
So do [=created=].

Reply:
Updated, agree with future conflicts.